client/components/VideoChat.jsx

```diff
@@ -15,7 +15,7 @@ class VideoChat extends React.Component {
 
   componentDidMount() {
     const constraints = {
-      audio: true,
+      audio: false, // ZS set this to false so that annyang can access the mic
```

Comment:
Yeah, I agree - I think being able to talk to your movie buddy is an important feature to keep, so if we can have both working in the final product that would be awesome :)